core/src/main/java/org/jdbi/v3/core/statement/ParsedSql.java

```diff
@@ -72,6 +72,25 @@ public String toString() {
             + '}';
     }
 
+    /**
+     * A static factory for {@link ParsedSql} object. It takes two
+     * arguments. The first one is SQL which must contain only
+     * positional parameters (i.e. the {@value #POSITIONAL_PARAM}
+     * characters). If your code contains named parameters (i.e.
+     * the ones followed by a colon) then you have to replace them
```

Comment:
aren't they _preceded_ by a colon? genuinely asking

Reply:
Yes, you are correct. My bad. Will fix this.

Reply:
Done.